protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing packets. " ..
+            "Enter value without leading <code>0x</code>.")
```

Comment:
AFAIK there is no easy way to limit an int to 32bit using the built-in datatypes of LuCI. The same goes for hexstring with leading `0x`. Not that I am happy with how this works, but there is no better way currently.

Let me see if I can add another datatype or if the maintainer have had enough of me adding various datatypes :)